components/user-store/org.wso2.carbon.identity.user.store.count/src/main/java/org/wso2/carbon/identity/user/store/count/UserStoreCountService.java

```diff
@@ -109,27 +111,24 @@
      * @return the number of users matching the given claim and filter by each domain
      */
     public PairDTO[] countClaim(String claimURI, String valueFilter) throws UserStoreCounterException {
+
         Set<String> userStoreDomains = UserStoreCountUtils.getCountEnabledUserStores();
         PairDTO[] claimCounts = new PairDTO[userStoreDomains.size()];
         int i = 0;
 
         for (String userStoreDomain : userStoreDomains) {
-            UserStoreCountRetriever counter = UserStoreCountUtils.getCounterInstanceForDomain(userStoreDomain);
-            Long count = Long.valueOf(-1);
-            if (counter != null) {
-                try {
-                    count = counter.countClaim(claimURI, valueFilter);
-                } catch (UserStoreCounterException e) {
-                    log.error("Error while getting user count with claim : " + claimURI + " from user store domain : "
-                            + userStoreDomain, e);
-                }
-            } else {
-                //no action
+            long count = -1L;
+            String filterWithDomain = getFilterWithDomain(userStoreDomain, valueFilter);
+
+            try {
+                count = getUserCountWithClaims(claimURI, filterWithDomain);
+            } catch (UserStoreCounterException e) {
+                log.error("Error while getting user count with claim : " + claimURI + " from user store domain : "
```

Comment:
Shall we add a comma after the parameter? Ex: `"Error while getting user count with claim : " + claimURI + ", from user`